### DISCUSSION TITLE
Saral Backend Performance Tuning and Load Testing Exercise proposal

### DISCUSSION THREAD

**Discussion by dileep-gadiraju** [Sunbird-Saral/Community#68] [Ideas]
Hi  @mathewjpallan/ @gandham-santhosh,

As discussed with you, below is the proposal for the Saral backend performance tuning exercise. 

![image](https://user-images.githubusercontent.com/89640648/229035445-f7cb8d4d-b2e9-426a-89b5-f266629ff006.png)

@sivaprakash123 / @vky25 / @harish-challa ,

@mathewjpallan gave the below recommendations.

-  Use the latest monitoring sunbird stack - grafana, Graylog  , please plan accordingly.
-  Use 8 CPU VMs for Mongodb and EKS clusters 

cc: @surendrasinghs

**Comment by surendrasinghs** [comment]
@sukhpreetsamagra @suresh12 @ChakshuGautam Hope you are already aligned and in-tune with this.
